modules/creating-manifest-file-customized-br-ex-bridge.adoc

```diff
@@ -136,12 +136,18 @@ spec:
           source: data:text/plain;charset=utf-8;base64,<base64_encoded_nmstate_configuration> <3>
         mode: 0644
         overwrite: true
-        path: /etc/nmstate/openshift/cluster.yml
+        path: /etc/nmstate/openshift/worker-0.yml <4>
+      - contents:
+          source: data:text/plain;charset=utf-8;base64,<base64_encoded_nmstate_configuration>
+        mode: 0644
+        overwrite: true
+        path: /etc/nmstate/openshift/worker-1.yml
 # ...
 ----
-<1> For each node in your cluster, specify the hostname path to your node and the base-64 encoded Ignition configuration file data for the machine type. If you have a single global configuration specified in an `/etc/nmstate/openshift/cluster.yml` configuration file that you want to apply to all nodes in your cluster, you do not need to specify the hostname path for each node. The `worker` role is the default role for nodes in your cluster. The `.yaml` extension does not work when specifying the hostname path for each node or all nodes in the `MachineConfig` manifest file.
+<1> For each node in your cluster, specify the hostname path to your node and the base-64 encoded Ignition configuration file data for the machine type.
```

Comment:
I think this note is on the wrong field. The hostname isn't relevant for the one this refers to currently.